packages/amplify-provider-awscloudformation/src/aws-utils/aws-s3.js

```diff
@@ -2,6 +2,10 @@ const aws = require('./aws.js');
 const _ = require('lodash');
 const providerName = require('../../lib/constants').ProviderName;
 const configurationManager = require('../../lib/configuration-manager');
+const fs = require('fs-extra');
+const ora = require('ora');
+
+const minChunkSize = 5 * 1024 * 1024; // 5 MB https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3/ManagedUpload.html#minPartSize-property
```

Comment:
Can we do some profiling of multipart upload for different sizes content and before selecting 5MB as the threshold for multipart upload

Reply:
https://docs.aws.amazon.com/AmazonS3/latest/dev/qfacts.html
I chose this because that's minimum size and don't really see that threshold being hit of. I can go only higher from 5 MB  up to 5 GB. We can still profile for higher sizes